source/math/FloatVectorList.ooc

```diff
@@ -181,16 +186,10 @@ FloatVectorList: class extends VectorList<Float> {
 		max != 0 ? (this / max) : this copy()
 	}
 	getOnes: static func (count: Float) -> This {
```

Comment:
I think we discussed calling these `ones` and `zeros`, respectively, but I guess it can wait.